kitura-basic.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
```

Comment:
Please add the IBM copyright header to every file in this project. It needs to go after the shebang.